tests/test_tokenization_auto.py

```diff
@@ -149,7 +149,7 @@ def test_tokenizer_identifier_with_correct_config(self):
     @require_tokenizers
     def test_tokenizer_identifier_non_existent(self):
         for tokenizer_class in [BertTokenizer, BertTokenizerFast, AutoTokenizer]:
-            with self.assertRaises(EnvironmentError):
+            with self.assertRaises(ValueError):
```

Comment:
Nice! Maybe even check that the "typo" error message is hit here with https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex ?

Reply:
Good idea, done!